app/src/main/res/values-sk/strings.xml

```diff
@@ -148,8 +148,8 @@ Sledujte svoj zdravotný stav. Viac informácií nájdete v aplikácii.</string>
     <string name="dashboard_positive_test_subtitle">Ak je výsledok pozitívny, upozornite ostatných.</string>
     <string name="dashboard_positive_test_button">Anonymne upozorniť ostatných</string>
     <string name="dashboard_travel_title">Spolupráca so zahraničím</string>
-    <string name="enabled">Zapnutá</string>
-    <string name="disable">Vypnutá</string>
+    <string name="enabled">Zapnuto</string>
```

Comment:
Slovenštinu neměň, to se musí updatnout v OneSky. V kódu se to pak přepíše exportem z OneSky.